portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,43 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+.carousel-inner img {
+    width: 100%;
+    height: 100%;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.section1{
```

Comment:
If you wanted to, you could also organize this as:
```
.section1, .section2 {
 // all common attributes
}
.section1 {
 background-color: #555555;
}
.section2 {
 background-color: #556699;
}
```

It just depends on if the common values are used in multiple places and would likely be updated together in the future.

There's never a "just one right way" to structuring this, what you have works great and is often better for a first version (before it's clear if the values really need to be shared or not).